EIPS/eip-crosschain-id.md

```diff
@@ -0,0 +1,131 @@
+---
+eip: <to be assigned>
+title: Crosschain Identifier Specification
+author: Weijia Zhang <weijia@wanchain.org>, Peter Robinson <peter.robinson@consensys.net>
+discussions-to:
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-10-21
+requires (*optional): <EIP number(s)>
+replaces (*optional): <EIP number(s)>
+---
+
+## Simple Summary
+
+Today chainid definition as specified in EIP-155 is an integer type.  This is not unique enough to identify all public and private blockchains.  A blockchain identifier (crosschain id) should be unique and satisfy the following requirements:
```

Comment:
Is this truly case? I believe `v` can be up to `2^256`. Since the chain id is multiplied by 2, that should leave `2^255` identifiers?

Reply:
Yes, chainid is currently a 32byte (256 bit) field with EIP-1344.   The spec of EIP-155 gave a perception that chain-id is an integer.  This spec is to give semantic meaning to this 256 bit/32 byte chainid. 
We can modify this as well.